packages/analytics-js-integrations/src/integrations/MicrosoftClarity/browser.js

```diff
@@ -55,7 +56,11 @@ class MicrosoftClarity {
     if (context?.traits?.customPageId) {
       customPageId = context.traits.customPageId;
     }
-    window.clarity('identify', userId, sessionId, customPageId);
+
+    const identifyPromise = window.clarity('identify', userId, sessionId, customPageId);
+    if (!!identifyPromise) { // Clarity SDK is ready
```

Comment:
To be more cautious, let's check for the promise type here.
```
if (typeof identifyPromise?.then === 'function')

Reply:
Done